### PR TITLE
2021 03 23 deeamon128 signup form

### PR DESCRIPTION
**Pull request by deeamon128**

### CODE REVIEW

File: web/deeamon128/html/signup-form/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/47ce19f1c7.js" crossorigin="anonymous"></script>
+  </script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
```
Comment (biggicode) [inline web/deeamon128/html/signup-form/index.html]: indentation

**Review by biggicode — APPROVED** [review]

File: web/deeamon128/html/signup-form/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/47ce19f1c7.js" crossorigin="anonymous"></script>
+  </script>
```
Comment (BogdanNegura) [inline web/deeamon128/html/signup-form/index.html]: you missed the start tag for "</script>"

**Review by BogdanNegura — APPROVED** [review]

File: web/deeamon128/html/signup-form/index.html (hunk 1 above)
Comment (deeamon128) [reply web/deeamon128/html/signup-form/index.html]: thank you

File: web/deeamon128/html/signup-form/index.html (hunk 2 above)
Comment (deeamon128) [reply web/deeamon128/html/signup-form/index.html]: my bad. it was from a previous code that I have deleted. thanks a lot.

File: web/deeamon128/html/signup-form/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <script src="https://kit.fontawesome.com/47ce19f1c7.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+  <title>Signup Form</title>
+</head>
+<body>
+  <form action="#" class="form">
+    <div class="icon"><i class="fas fa-times"></i></div>
+    <h1>Join our community of photography lovers from all over the world</h1>
+    <div class="inputs">
```
Comment (robertoalx) [inline web/deeamon128/html/signup-form/index.html:17]: for clarity here, you can choose a better class name as an example, "email-wrapper"

File: web/deeamon128/html/signup-form/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      <label for="email">email</label>
+      <input id="email" type="email" placeholder="email"> 
+    </div>
+    <div class="inputs">
```
Comment (robertoalx) [inline web/deeamon128/html/signup-form/index.html:21]: Same here "password-wrapper" and so on

File: web/deeamon128/html/signup-form/style.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+#email::placeholder {
+  width: 100%;
+  border: none;
+  background: none;
+  outline: none;
+  color: rgba(255, 255, 255, 0.6);
+  margin-top: 0.5rem;
+  font-size: 0.8rem;
+  padding-bottom: 0.5rem;
+}
```
Comment (robertoalx) [inline web/deeamon128/html/signup-form/style.css:67]: empty line here

File: web/deeamon128/html/signup-form/style.css [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+#password::placeholder {
+  width: 100%;
+  border: none;
+  background: none;
+  outline: none;
+  color: rgba(255, 255, 255, 0.6);
+  margin-top: 0.5rem;
+  font-size: 0.8rem;
+  padding-bottom: 0.5rem;
+}
```
Comment (robertoalx) [inline web/deeamon128/html/signup-form/style.css:79]: an empty line here as well

File: web/deeamon128/html/signup-form/index.html (hunk 3 above)
Comment (deeamon128) [reply web/deeamon128/html/signup-form/index.html:17]: instead of which class you mean?

**Review by robertoalx — APPROVED** [review]
Good job :)

File: web/deeamon128/html/signup-form/style.css (hunk 6 above)
Comment (deeamon128) [reply web/deeamon128/html/signup-form/style.css:79]: updated. thanks

**Comment by adelacecilia** [comment]
Looks good! Keep it up! 💯 😺

**Review by marianzburlea — COMMENTED** [review]
💯

**Pull request merged**